Installers/WordPress.sh

```
@@ -0,0 +1,131 @@
+#/bin/sh
```

Shellcheck (0.11.0)

[error] Use #!, not just #, for the shebang.

(SC1113)